build/presubmit.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- name: "gcr.io/cloud-builders/go"
+  entrypoint: "sh"
```

Comment:
Why not just make the entrypoint be the check script?

Reply:
That doesn't seem to work. It gives me `standard_init_linux.go:211: exec user process caused "exec format error"`